CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## HEAD
 
+* [minimagick] Disallow splitting multi-layer images into multiple single-layer images (@janko-m)
```

Comment:
I would tweak this slightly:

[minimagick] Disallow splitting multi-layer images into multiple single-layer images by default to prevent weird exceptions with multi-layer PDF images. Splitting can be re-enabled by passing the `:allow_splitting` option to `.saver()`. (@janko-m)